src/components/ButtonPanel.js

```diff
@@ -1,42 +1,51 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-const ButtonPanel = () => (
-  <div className="flex flex-col btnPanel">
-    <div className="rowPanel">
-      <Button color="#d3d3d3" buttonName="AC" />
-      <Button color="#d3d3d3" buttonName="+/-" />
-      <Button color="#d3d3d3" buttonName="%" />
-      <Button buttonName="÷" />
-    </div>
+const ButtonPanel = props => {
+  const handleClickBP = buttonName => props.clickHandler(buttonName);
```

Comment:
- [ ] [OPTIONAL] By convention we call this function that handles a click event `handleClick`, so you can drop the `BP` unless you have another function in this same file that has the same name.